### PR TITLE
Core generator

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: src/ResolverManager.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  private buildRead(gqlType: Type): IResolverFormat {
+    return {
+      fieldName: gqlType.name,
+      action: ResolverType.READ,
+      // Needs fix
+      implementation: `${this.knexContext}.select().from('${this.prefix}${gqlType.name.toLowerCase()}')`
+    };
+  }
+
+  // Notes: Server side needs to be refactored to implement this properly
```
Comment (wtrocki) [inline src/ResolverManager.ts]: ping @darahayes

File: src/schema/Helpers.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ * Contains helpers used in all templates
+ */
+export class HandlebarsHelpers {
+
+  public registerHelpers = () => {
+    // tslint:disable-next-line:cyclomatic-complexity no-any
+    handlebars.registerHelper('ifCond', function(v1: any, operator: string, v2: any, options: any) {
+      switch (operator) {
+        case '==':
+          return v1 === v2 ? options.fn(this) : options.inverse(this);
```
Comment (darahayes) [inline src/schema/Helpers.ts]: Nice stackoverflow code 😉

Comment (wtrocki) [reply src/schema/Helpers.ts]: I actually getting all of this from exsting generator: 
https://github.com/dotansimha/graphql-code-generator/blob/master/packages/graphql-codegen-compiler/src/handlebars-extensions.ts

This generator proven to be overkill so I'm not using it, but there is tons of inspiration in current code.

Comment (darahayes) [reply src/schema/Helpers.ts]: https://stackoverflow.com/a/16315366/8511930 😃

**Comment by wtrocki** [comment]
FYI @StephenCoady

**Comment by wtrocki** [comment]
Merging for now to master for testing purposes.

**Pull request merged**